### ISSUE TITLE
No media files error

### ISSUE THREAD

**Issue by Steven-Biro** [Laharah/deluge-FileBotTool#44]
I'm having some trouble with torrents that do not contain recognizable files, most of my torrents moves fine, but here is an example of a movie with extra files included beyond that video file and nfo.
This one contains a Screenshots folder with example screenshots of the video quality, as well as a cover image and a txt file with details on the video such as the format, bit rate, etc..

Is there a way to handle this? I've looked around through all the documentation here and couldn't find anything.

Deluge info:
```Client: 1.3.15
Server: 1.3.15
libtorrent: 1.1.11.0
```
FileBot plugin version: 1.2.7

FileBot plugin error:
```
Deluge info:
```Client: 1.3.15
Server: 1.3.15
libtorrent: 1.1.11.0
```
FileBot plugin version: 1.2.7

FileBot plugin error:
```
FilebotRuntimeError error on torrent 1f08***********************88c2:
    FILEBOT OUTPUT DUMP:
    Rename movies using [TheMovieDB]
    No media files: [/path/torrnet_name/MediaInfo.txt, /path/torrnet_name/Screenshots/01.png, /path/torrnet_name/Screenshots/02.png, /path/torrnet_name/Screenshots/03.png, /path/torrnet_name/Screenshots/04.png, /path/torrnet_name/Screenshots/05.png, /path/torrnet_name/Screenshots/06.png, /path/torrnet_name/Screenshots/07.png, /path/torrnet_name/Screenshots/08.png, /path/torrnet_name/cover.png]
    Failure (×_×)⌒☆

    stderr:
```

format expression: `{kodi}`

**Comment by Laharah** [comment]
It looks like filebot is being run on a torrent doesn't have a video file. Either the video isn't in the torrent, or the file priority on the video file was set to "Do Not Download". FilebotTool does not send files marked do not download to FIlebot because it presumes that they won't exist. If Filebot gets told to sort a file that isn't there it'll throw an error and halt. 

So long as the extras etc. are in the same torrent, filebot will ignore them and the tool will everything extra along for the ride to wherever filebot sends the media files. But if there isn't anything filebot can sort, the tool doesn't know where to send everything else.

If you've got a torrent that doesn't have any media files in it, you'll have to move it manually. You'll have to right click > Move Storage and then enter `/path/to/movie/folder`.

**Comment by Steven-Biro** [comment]
There is a video file in the directory, I just opened it on vlc from my phone over the local network, skipped to the mid point, then to the end. I've also done a forced recheck on the torrents

Here is the structure of one of the torrents 

Torrent_name:
* cover.png
* MediaInfo.txt
* Screenshots
* Torrent_name.mkv
* Torrent_name.nfo


Another of the torrents looked like this:

Torrent_name:
* Sample.mkv
* Torrent_name.mkv
* Torrent_name.nfo

I double checked all the other torrents moved correctly and they did not have any extra files with vague names, 


I just went to find a random torrent from a public tracker that has extra files in it to test:

![torrent](https://user-images.githubusercontent.com/14033941/77236596-82af1480-6b96-11ea-8ddb-09727a85c866.png)

**Comment by Laharah** [comment]
Filebot was updated about a week ago. This looks like a change to the way filebot handles extra found files. I'll open a bug report there and then either I'll change the way FileBotTool handles files, or there will be a fix put into filebot and you'll have to update.

I'll keep you posted, but in the meantime I know the behavior works as expected with filebot 4.8.5

**Comment by Laharah** [comment]
Turns there was a bug in filebot. You can work around it by selecting a database to use (thetvdb, themoviedb, etc.)

The writer of filebot has put a fix for the issue in FileBot 4.9.0 (r7255), which you can download from the beta branch here:

https://get.filebot.net/filebot/BETA/

It will probably be released into the main branch in a day or two, if you'd rather wait until then. So either update filebot to the beta or select a database when you're sorting till then.

**Comment by Steven-Biro** [comment]
Thanks a bunch for the help debugging this!
